BubbleRankingIndicator.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
might wanna give this PR a once over and make sure we are only versioning what we want to. is this needed?

Reply:
I can do that.